projects/components/src/form/form-input/form-input.component.html

```diff
@@ -17,18 +17,20 @@
                     [min]="min"
                     [max]="max"
                     [attr.maxlength]="maxlength"
+                    [attr.aria-required]="showAsterisk"
+                    [attr.aria-describedby]="showErrors ? errorsId : descriptionId"
```

Comment:
descriptionId is valid only if there is a description

Reply:
It appears we always have a value for `description`. In the `BaseFormControl` the `@Input` is initialized: 

```
/**
 * The description that appears below the control
 * \u00A0 is &nbsp; (non-breaking space)
 */
@Input() description = '\u00A0';
```